templates/docs/base/code.md

```diff
@@ -52,31 +52,27 @@ A code snippet block can include an icon by using the `.p-code-snippet__block--c
 View example of the code snippet
```

Comment:
Missed that before, there is a typo in class name `.p-code-snippet__block--code`, should be `--icon`.